acceptance/bundle/plan/no_upload/databricks.yml

```diff
@@ -0,0 +1,20 @@
+bundle:
```

Comment:
For more comprehensive test, can we add "bundle plan" + request recording to default-python various local and integration tests and check that there are no side effects there?